examples/jss/src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { render } from 'react-dom';
 
-import './global-styles';
+import GlobalStyles from './components/global-styles';
 import App from './components/app';
 
-render(<App />, document.getElementById('example-root'));
+render(<GlobalStyles><App /></GlobalStyles>, document.getElementById('example-root'));
```

Comment:
This is still hacky, if this would just work with

```JS
import './global-styles';
```

without having to add an unnecessary wrapper component I'd be a fan…